SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -670,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
Is this required? No sense to bump deployment target if it can still work on `7.0`.